Xamarin.Forms.Platform.UAP/FormsTextBoxStyle.xaml

```diff
@@ -173,6 +173,7 @@
 						                  Foreground="{ThemeResource SystemControlForegroundBaseHighBrush}" FontWeight="Normal"
 						                  Margin="0,0,0,8" Grid.Row="0" Visibility="Collapsed" x:DeferLoadStrategy="Lazy" />
 						<ScrollViewer x:Name="ContentElement" AutomationProperties.AccessibilityView="Raw"
+                                      Foreground="{TemplateBinding ForegroundFocusBrush}"
```

Comment:
The code on [L141-143](https://github.com/xamarin/Xamarin.Forms/blob/bad77d0ee4b9d3a2b642cbc3c35d87e848d5ece3/Xamarin.Forms.Platform.UAP/FormsTextBoxStyle.xaml#L141-L143) of this template seems like it could be removed and not make a difference, but I left it for the time being.